### PR TITLE
drivers/{l, m, n}*: add modules to Kconfig

### PR DESCRIPTION
**Pull request by leandrolanzieri**
### Contribution description
This models more driver modules as Kconfig symbols and adds the configurations for their applications. The following drivers are included:

- l3g4200d
- lc709203f
- lis2dh12
- lis3dh
- lis3mdl
- lpd8808
- lpsxxx
- lsm6dsl
- lsm303dlhc
- ltc4150
- mag3110
- mcp2515
- mhz19
- mma8x5x
- mma7660
- motor_driver
- mpl3115a2
- mpu9x50
- mq3
- mtd_flashpage
- mtd_mapper
- cpu/lpc23xx/mci
- mtd_mci
- mtd_sdcard
- mtd_spi_nor
- ncv7356
- nvram
- nvram_spi

### Testing procedure
- Check that the dependencies are correctly modelled
- Check that the new symbols are only visible and usable when `TEST_KCONFIG=1`
- Check that the correspondent applications for the modelled drivers were added to the CI's checks
- Green CI

### Issues/PRs references
None

### CODE REVIEW

File: drivers/mtd_mci/Kconfig [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_MTD_MCI
+    bool "MTD interface for LPC23XX MCI"
+    depends on TEST_KCONFIG
+    depends on MODULE_MCI
```
Comment (fjmolinas) [inline drivers/mtd_mci/Kconfig:11]:
```suggestion
    select MODULE_MTD
    depends on MODULE_MCI
```

**Review by fjmolinas — COMMENTED** [review]
One comment, please squash and trigger CI once addressed

File: drivers/mtd_mci/Kconfig (hunk 1 above)
Comment (leandrolanzieri) [reply drivers/mtd_mci/Kconfig:11]: Nice catch @fjmolinas ! Added

**Review by fjmolinas — COMMENTED** [review]
ACK!

**Review by fjmolinas — APPROVED** [review]
ACK, verified content matches, murdock tested the rest.

**Comment by leandrolanzieri** [comment]
Thanks for the thorough review!

**Pull request merged**